base/subarray.jl

```diff
@@ -82,25 +95,29 @@ julia> parent(s_a)
 ```
 """
 parent(V::SubArray) = V.parent
-parentindexes(V::SubArray) = V.indexes
+parentindexes(V::SubArray) = V.indices
```

Comment:
This seems increasingly weird as an exported function. Should we just get rid of it?

Reply:
I definitely agree! I had it on the chopping block but a handful of packages use it a surprising amount of times. I'll want to see how they use it first.